security/security.manager.ts

```diff
@@ -0,0 +1,111 @@
+import { HttpBadRequestError } from '@errors/http';
+import { getEnv } from '@helper/environment';
+import { IconikToken, IconikTokenModel, IconikTokenSchema } from '@models/DynamoDB/iconik-token.model';
+import { IconikService } from '@workflowwin/iconik-api';
+import { MetadataFieldSchema } from '@workflowwin/iconik-api/dist/src/metadata/metadata-methods';
+import { CustomActionSchema } from '@workflowwin/iconik-api/src/assets/assets-methods';
+import { WebhookResponseSchema } from '@workflowwin/iconik-api/src/notifications/notifications-methods';
+import { ScanResponse } from 'dynamoose/dist/DocumentRetriever';
+import { SecurityService } from './security.service';
+
+const refreshHoursField: MetadataFieldSchema = {
+  field_type: 'integer',
+  name: 'Set Token Refresh Time',
+  description: 'WIN Automation View',

```

Comment:
I think there should be `label` field with `win_SetTokenRefreshTime`. Am i right?

Reply:
It should look like this
![image](https://user-images.githubusercontent.com/26054907/119792176-2a804a80-bede-11eb-946f-3f91aeb7e4fd.png)
@emahota-flo Check all your fields